projects/vendor/README.md

```diff
@@ -0,0 +1,24 @@
+# Vendor
```

Comment:
Could you update this with a quick descriptions of the steps to use this. No need to be super polished our writer will edit it as long as there is the basic.

Reply:
done basically